libraries/cyclestreets-view/src/main/java/net/cyclestreets/views/CycleMapView.java

```diff
@@ -74,13 +77,13 @@ public CycleMapView(final Context context, final String name, final Fragment fra
     boolean hasWritePermission = PermissionsKt.hasPermission(context, Manifest.permission.WRITE_EXTERNAL_STORAGE);
     Configuration.getInstance().load(context, PreferenceManager.getDefaultSharedPreferences(context));
     Log.i(TAG, "Creating map view. App has write-external permission? " + hasWritePermission +
-        "; osmdroid base path: " + Configuration.getInstance().getOsmdroidBasePath().getAbsolutePath() +
-        "; osmdroid tile cache location: " + Configuration.getInstance().getOsmdroidTileCache().getAbsolutePath());
+            "; osmdroid base path: " + Configuration.getInstance().getOsmdroidBasePath().getAbsolutePath() +
+            "; osmdroid tile cache location: " + Configuration.getInstance().getOsmdroidTileCache().getAbsolutePath());
 
     mapView_ = new MapView(context, TileSource.mapTileProvider(context));
     addView(mapView_);
 
-    prefs_ = context.getSharedPreferences("net.cyclestreets.mapview."+name, Context.MODE_PRIVATE);
+    prefs_ = context.getSharedPreferences("net.cyclestreets.mapview." + name, Context.MODE_PRIVATE);
 
     mapView_.getZoomController().setVisibility(CustomZoomButtonsController.Visibility.NEVER);
```

Comment:
nhiều changes là do format nhỉ, chú ý những dự án maintaince cần để ý cái này nha, đừng format toàn bộ file mà chỉ những cho mình thay đổi thôi em nhé. Nếu k rất khó review

Reply:
Các change ở trên là do format thôi ạ. Lần sau em sẽ check lại cái này